selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/DefaultAndroidEmulator.java

```diff
@@ -494,6 +500,8 @@ public void stop() throws AndroidDeviceException {
           try {
             stopEmulator();
           } catch (AndroidDeviceException sce) {
+	    removeLockFiles();
```

Comment:
also there's a 'stopEmluator' call earlier in the method that could potentially throw the AndroidDeviceException too now. You'll want to catch it there too.